response_operations_ui/templates/update-event-date.html

```diff
@@ -1,128 +1,208 @@
-      </div>
-      {%- endif -%}
+              {% endfor %}
+              <h2 class="u-fs-m" id='error-saving-message'>Error updating {{event_name}} </h2>
+              <div id='error-text'>
+                {{
+                    onsList({
+                        "classes": "list--bare",
+                        "itemsList": errorData
+                    }) 
+                }}
+              </div>
+            {% endif %}
+          {% endcall %}
+        {% endif %}
       {%- endwith -%}
+      <h1 class="u-fs-xl u-mt-m">Change business event</h1>
+      {{ 
+        onsMetadata({
+          "termCol": "2",
+          "descriptionCol": "10",
+          "itemsList": [
+            {
+              "term": "Survey:",
+              "descriptions": [ 
+                  {
+                      "description": survey.surveyRef ~ ' ' ~ survey.shortName
+                  }
+              ]
+            },
+            {
+              "term": "Period:",
+              "descriptions": [ 
+                  {
+                      "description": ce.exerciseRef,
+                      "id": "period"
+                  }
+              ]
+            }
+          ]
+        })
+      }}
 
-    {% if date_restriction_text %}
-      {% for text in date_restriction_text %}
-      <div>
-        <p> {{ text }} </p>
-      </div>
-      {% endfor %}
-    {% endif %}
+      <h2 class="u-fs-l u-mt-l">{{event_name}}</h2>
+      {% if date_restriction_text %}
+        {% for text in date_restriction_text %}
+        <div>
+          <p> {{ text }} </p>
+        </div>
+        {% endfor %}
+      {% endif %}
 
     <form method="post" class="">
       <div class="date-time-editor u-mb-l">
         {{ form.csrf_token }}
-        <fieldset class="fieldgroup fieldgroup--inline fieldgroup--date" data-qa="widget-date" role="group">
-          <legend class="fieldgroup__title u-fs-r--b">Date</legend>
-          <div class="fieldgroup__fields">
-            <div class="field field--input">
-              <label class="label u-fs-r" for="date-en-gb-day" id="label-date-range-from-day" data-qa="label-day">Day</label>
-              {{ form.day(id="date-en-gb-day", size=2, class_="input input--text input--StringField input--w-2", type="number") }}
-            </div>
-            <div class="field field--select">
-              <label class="label u-fs-r" for="date-en-gb-month" id="label-date-range-from-month" data-qa="label-month">Month</label>
-              {{ form.month(id="date-en-gb-month", class_="input input--select input--w-auto") }}
-            </div>
-            <div class="field field--input">
-              <label class="label u-fs-r" for="date-en-gb-year" id="label-date-range-from-year" data-qa="label-year">Year</label>
-              {{ form.year(id="date-en-gb-year", class="input input--text input--StringField input--w-4", type="number") }}
-            </div>
-          </div>
-        </fieldset>
+        {{
+            onsDateInput({
```

Comment:
Currently doesn't work right. On an edit screen it should load up the old values and in this one it doesn't do that.
Moreover, when defaults are put in place, the time values don't work as they're select values don't act right when given a string as they're SelectFields instead of StringFields.

Reply:
Actually SelectField vs Stringfield thing was a mistake I made in the template, but the point still stands that it doesn't replay current values if available here (and I imagine everywhere else this change was made)